@xen-orchestra/build-template/src/index.spec.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-env jest */
+import buildTemplate from '.'
+
+it('builds a template', () => {
+  const replacer = buildTemplate('{name}_\\{name}_\\\\{name}_{id}_%_FOO', {
+    '{name}': obj => obj.name,
+    '{id}': 1235,
```

Comment:
It's a bit hard to understand which one is going to come from the object and which one is constant. So maybe call them `{property}` and `{constant}` for this test?